newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -59,11 +72,20 @@ export const AssetCard = ({ assetShortHeader, onOpenDetails, size }: Props) => {
           <CheckeredBackground />
           <CorsAwareImage
             key={assetShortHeader.previewImageUrls[0]}
-            style={{
-              ...styles.previewImage,
-              maxWidth: 128 - 2 * paddingSize,
-              maxHeight: 128 - 2 * paddingSize,
-            }}
+            style={
+              isPixelArt(assetShortHeader)
+                ? {
+                    ...styles.previewImage,
+                    ...styles.previewImagePixelated,
```

Comment:
Make a ternary inside this:
...(isPixelArt ? styles.previewImagePixelated : undefined),

Reply:
(Shorter and avoid to duplicate the max width/height, and same number of objects creation)